BlockServer/test_modules/runcontrol_manager_tests.py

```diff
@@ -48,19 +48,24 @@ def add_block(cs, data):
     cs.add_block(data)
 
 
+def dummy_sleep_func(seconds):
+    return
+
+
+def _get_time(t, **kwargs):
```

Comment:
The name of this function isn't quite descriptive enough for me. `_get_relative_time` might be more appropriate